yt_dlp/postprocessor/metadataparser.py

```diff
@@ -1,29 +1,25 @@
 import re
-from enum import Enum
 
 from .common import PostProcessor
+from ..utils import Namespace
 
 
 class MetadataParserPP(PostProcessor):
-    class Actions(Enum):
```

Comment:
Actually, this one was fine. The fact that doing `assert isinstance(action, self.Actions)`, as you did later, makes sense makes it a proper type.

Reply:
I never liked how the function was decided using `getattr(self, action.value)`